docs/engineering/android-app/configuring/working-with-rules.mdx

```diff
@@ -89,3 +89,21 @@ This rule is used within the ruels engine or workflow to generate the correct se
       ]
     }
 ```
+
+## Using the `extractPractitionerInfoFromSharedPrefs()` rule
+
+This rule is used within the rules engine or workflow to extract `PractitionerId`, `PractitionerCareTeam`, `PractitionerOrganization` and `PractitionerLocation` from `Shared Preference`.
```

Comment:
List all the enums that are supported by this change e.g in order to fetch `PractitionerId` which enum would be used?

A tabular format would be easy to read.

Reply:
Sure. I have done so